scripts/cuvslam_kpi_report.py

```
@@ -0,0 +1,447 @@
+        return None
+
+    all_stats = parse_all_stats_json(all_stats_json)
+    if not all_stats:
+        print(f'Warning: failed to parse all_stats.json in {stats_folder}')
+        return None
+
+    if all_stats and 'odometry_mode' in all_stats[0]:
+        dataset_type = odometry_mode_to_type(all_stats[0]['odometry_mode'])
+        print(f'  Detected dataset type: {dataset_type} (from odometry_mode: {all_stats[0]["odometry_mode"]})')
+    else:
+        print(f'  Warning: odometry_mode not found in JSON, falling back to folder name parsing')
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)